openapi/components/schemas/CheckoutForm.yaml

```
@@ -16,3 +16,12 @@ allOf:
           - active
           - inactive
         default: active
+      _links:
+        type: array
+        description: The links related to resource.
```

Nitpick: Where possible, we should avoid starting descriptions with "the"

It can make the docs tiresome to read

```suggestion
        description: Links associated with the resource.
```